compiler/src/iree/compiler/Codegen/LLVMGPU/TransformExtensions/LLVMGPUExtensionsOps.td

```diff
@@ -682,4 +682,41 @@ def PackSharedMemoryAllocOp :  Op<Transform_Dialect, "iree.pack_shared_memory_al
   }];
 }
 
+def TestAMDGPUContractionDistribution :
```

Comment:
Flyby comment, we should later just make this TestAMDGPUDistributionPatterns or something. Nothing to do for now.

Reply:
sounds good